Kitodo-LongTimePreservationValidationModule/src/main/resources/jhove.conf

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<jhoveConfig version="1.0"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xmlns="http://hul.harvard.edu/ois/xml/ns/jhove/jhoveConfig"
+ xsi:schemaLocation="http://hul.harvard.edu/ois/xml/ns/jhove/jhoveConfig
+                     http://hul.harvard.edu/ois/xml/xsd/jhove/jhoveConfig.xsd">
+  <jhoveHome>src/test/resources</jhoveHome>
```

Comment:
This value `src/test/resources` is only valid for test execution and must be adjusted in normal environment or? If so why did you not add this kind of file to `/src/test/resources` and use it inside the tests?